src/main/java/com/kodilla/ecommercee/controller/GroupController.java

```diff
@@ -1,50 +1,43 @@
 package com.kodilla.ecommercee.controller;
 
 import com.kodilla.ecommercee.dto.GroupDto;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.PutMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.kodilla.ecommercee.mapper.GroupMapper;
+import com.kodilla.ecommercee.service.DbService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.ArrayList;
 import java.util.Arrays;
```

Comment:
Po usunięciu przykładowych danych te dwa importy są zbędne.
Jeżeli robisz commity z IntelliJ warto ustawić sobie opcję `Optimize imports` w okienku `Commit Changes` - wtedy przed wysłaniem kodu do repozytorium, IntelliJ m.in. usunie zbędne importy.
![obraz](https://user-images.githubusercontent.com/49949209/66341450-9ac40c00-e947-11e9-81fa-a74e83f074f7.png)